content/knowledge-base/lifecycle.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Product Lifecycle
+---
+
+## Software List
+
+Upbound provides support and software maintenance to paid customers. Supported software products include:
+
+* [UXP]({{< ref "uxp/_index.md" >}})
+* [Official Providers]({{< ref "upbound-marketplace/providers/_index.md" >}})
+
+## Maintenance and Updates
+
+Upbound supports Generally Available releases for up to 12 months. 
+All software follows semantic versioning of `X.Y.Z`.
+
+* A major release is a change in the first digit and usually indicates breaking API changes or, in the case of 1.0.0, indicates the first release where the API is considered complete and stable.
+* A minor release is a change in the second digit and indicates new features were added. It could also include new bug fixes that were not included in a previous patch release.
+* A patch release is a change in the third digit and only contains bug fixes, no new features.
+
+### UXP Schedule
+
+UXP follows the [Crossplane Release Cycle](https://docs.crossplane.io/knowledge-base/guides/release-cycle/). UXP has a major release every 3 months. Minor patch releases are available as needed. Upbound provides support for the current + 2 previous minor releases.
+
+### Official Provider Schedule
+
+Official providers are released at least monthly. 
```

Comment:
nit: sounds like an unnecessary promise. "Regular schedule is monthly".

Reply:
we release way more than monthly - 2 or 3x/month on average. This is sandbagged.

Reply:
That is correct. I am however actively steering us to where we only do this monthly (last Thursday of the month). The exception is if we have customers who need a fix urgently, we will cut an out-of-schedule release of a provider for them.